src/main/java/org/sunbird/common/util/Constants.java

```diff
@@ -465,7 +465,9 @@ public class Constants {
 	public static final String PASSBOOK_READ_API = "user.passbook.read";
 	public static final String PASSBOOK_ADMIN_READ_API = "user.passbook.admin.read";
 	public static final String PASSBOOK_ADD_API = "user.passbook.add";
-	public static final String USER_PASSBOOK_TABLE = "user_passbook";
+	public static final String USER_PASSBOOK_TABLE = "user_passbook_v2";
+
```

Comment:
Remove unnecessary line space

Reply:
Can i name table as user_passbookV2?
For now i have created table named as user_passbook_v2 in stage and in prod